src/views/Home/Home.tsx

```diff
@@ -35,11 +39,11 @@ export const Home: FunctionComponent = () => {
 
         <Info />
 
-        {fullSite ? <Categories /> : <></>}
+        {hasPackages && fullSite ? <Categories /> : <></>}
 
-        {fullSite ? <CDKTypeTabs /> : <></>}
+        {hasPackages && fullSite ? <CDKTypeTabs /> : <></>}
 
-        {fullSite ? <Featured /> : <></>}
+        {hasPackages && fullSite ? <Featured /> : <></>}
```

Comment:
For some reason it won't let me make a suggestion here, but can we clean this up to be

```tsx
{hasPackages && fullSite && (
  <>
    <Categories />
    <CDKTypeTabs />
    <Featured />
  </>
)}
```

Reply:
Done